examples/common/pigweed/rpc_services/Attributes.h

```diff
@@ -207,6 +228,15 @@ class Attributes : public pw_rpc::nanopb::Attributes::Service<Attributes>
         }
 
         app::AttributeValueDecoder decoder(tlvReader.value(), subjectDescriptor);
+
+        // Try to write using a custom Accessor first
```

Comment:
This re-states the code. Is there something we can say instead to augment the code rather than re-reading it? If not, lets drop the comment.